src/templates/partials/exportEnum.hbs

```diff
@@ -9,9 +9,25 @@
 {{/if}}
  */
 {{/ifdef}}
+{{#if @root.$config.useLegacyEnums}}
+export enum {{{name}}} {
+	{{#each enum}}
+	{{#if x-enum-description}}
+	/**
+	 * {{{escapeComment x-enum-description}}}
+	 */
+	{{else if description}}
+	/**
+	 * {{{escapeComment description}}}
+	 */
+	{{/if}}
+	{{{enumKey value x-enum-varname}}} = {{{enumValue value}}},
+	{{/each}}
+}
+{{else}}
 export type {{{name}}} = {{{enumUnionType enum}}};
 {{/if}}
-
+{{/if}}
 {{#if @root.$config.enums}}
```

Comment:
Add back one newline above this

Reply:
@mlankamp  Please add back one newline, run `npm run test:update` and then add the changeset as mentioned above, I cannot modify this PR. Then I will merge and release